### PR TITLE
Style Login and Signup forms along with Home page

### PR DESCRIPTION
**Pull request by sderacy**
This does pretty much what the PR says. This integrates bootstrap with all the code Payton wrote and I fixed some of the meta tags.  I didn't end up adding any CSS pages for the signup or login page because bootstrap is a powerful thing but run this on your side and make sure this checks out

### CODE REVIEW

**Comment by sderacy** [comment]
I almost forgot to put this but considering the spin up time for this, the login and registration works really well. However, I think there are some things that we can fix to make it a little more enjoyable for the user.

These are areas for improvement I found while running it during my styling process:

- We should ask user to repeat password for authentication, make sure they entered password correctly

- When there's an error signing up, it'd be nice if we could save some of the data

The home page looks like trash right now btw but I wasn't sure if I should get rid of the debug statements

File: views/account/signup/index.ejs [hunk 1]
```diff
@@ -5,20 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Sign Up</title>
+
+    <!-- Bootstrap links to make it work  -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet" />
   </head>
   <body>
-    <p class="message"><%= error %></p>
```
Comment (paytonshaltis) [inline views/account/signup/index.ejs:10]: Error message that blends in on signup page.

File: views/account/login/index.ejs [hunk 2]
```diff
@@ -5,16 +5,50 @@
...
+    style="background-image: url('/gooncardbackground.jpg');">
+    <div class="mask d-flex align-items-center h-100 gradient-custom-3">
+      <div class="container h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+          <div class="col-12 col-md-9 col-lg-7 col-xl-6">
+            <div class="card bg-dark" style="border-radius: 15px;">
+              <div class="card-body p-5">
+                <h2 class="text-uppercase text-center text-white mb-5"> Log In </h2>
+    
+                <p class="message"><%= error %></p>
```
Comment (paytonshaltis) [inline views/account/login/index.ejs:24]: Error message that blends in on login page.

File: views/account/signup/index.ejs [hunk 3]
```diff
@@ -5,20 +5,61 @@
...
+    style="background-image: url('/gooncardbackground.jpg');">
+    <div class="mask d-flex align-items-center h-100 gradient-custom-3">
+      <div class="container h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+          <div class="col-12 col-md-9 col-lg-7 col-xl-6">
+            <div class="card bg-dark" style="border-radius: 15px;">
+              <div class="card-body p-5">
+                <h2 class="text-uppercase text-center text-white mb-5"> Create an account</h2>
+  
+                <p class="message"><%= error %></p>
```
Comment (paytonshaltis) [inline views/account/signup/index.ejs:24]: Error message that blends in on signup page.

**Comment by paytonshaltis** [comment]
> I almost forgot to put this but considering the spin up time for this, the login and registration works really well. However, I think there are some things that we can fix to make it a little more enjoyable for the user.
> 
> These are areas for improvement I found while running it during my styling process:
> 
> * We should ask user to repeat password for authentication, make sure they entered password correctly
> * When there's an error signing up, it'd be nice if we could save some of the data
> 
> The home page looks like trash right now btw but I wasn't sure if I should get rid of the debug statements

- Definitely agree with two password fields. I just did one initially for testing account creation.
- Also agree with keeping fields. When login or signup fails, we can have the page load with the previous values.

I'll add these as two issues so we don't forget about them later on.

File: views/account/signup/index.ejs (hunk 3 above)
Comment (sderacy) [reply views/account/signup/index.ejs]: Really good catch, it should be fixed in the latest commit

File: views/home/index.ejs [hunk 4]
```diff
@@ -1,28 +1,70 @@
...
+          <ul>
+            <li>First Name: <%= user.first_name %></li>
+            <li>Last Name: <%= user.last_name %></li>
+            <li>Email: <%= user.email %></li>
+            <li>Password: <%= user.password %></li>
+          </ul>
+
 
     <!-- Testing requesting an image. -->
     <img src="/hamster.jpg" alt="Hamster image" />
```
[the remark this answers is not in the document]
Comment (sderacy) [reply views/home/index.ejs]: It has been obliterated.

**Review by paytonshaltis — APPROVED** [review]
Fantastic work chief.

**Comment by sderacy** [comment]
You too with the server code!

**Pull request merged**